lib/excon/test/plugin/server/puma.rb

```diff
@@ -4,7 +4,7 @@ module Plugin
       module Server
         module Puma
           def start(app_str = app, bind_uri = bind)
-            bind.host = bind_uri.host.gsub(/[\[\]]/, '')
+            #bind.host = bind_uri.host.gsub(/[\[\]]/, '')
```

Comment:
Is this supposed to still be commented out? Would be good to either uncomment, add an explanation as to why it is commented, or just delete it altogether if it is no longer needed.